tools/scripts/update_exp_files.sh

```diff
@@ -4,12 +4,12 @@ set -e
 COMMAND_FILE=$(mktemp)
 trap 'rm -f "$COMMAND_FILE"' EXIT
 
-# TODO(JEZ) Make this faster and better so we can uncomment this
```

Comment:
How much slower does this make update_exp_files.sh? I think that fundamentally `srb init` is slow regardless of bazel, and we probably still want to keep "updating testdata tests" and "updating srb init tests" separate.

Reply:
It's slow, but everything is running in parallel, so it didn't feel slower than normal. I'll try a run on `master` and this branch, and see what the difference is.